bookkeeper-http/http-server/src/main/java/org/apache/bookkeeper/http/HttpRouter.java

```diff
@@ -45,6 +45,7 @@
     public static final String LAST_LOG_MARK                = "/api/v1/bookie/last_log_mark";
     public static final String LIST_DISK_FILE               = "/api/v1/bookie/list_disk_file";
     public static final String EXPAND_STORAGE               = "/api/v1/bookie/expand_storage";
+    public static final String TRIGGER_GC                   = "/api/v1/bookie/trigger_gc";
```

Comment:
I think it is better to just call it `/api/v1/bookie/gc`. a `PUT` method is to trigger gc and a `GET` method is to query the gc status. we can leave implementing `GET` in a separate pull request. but at least it allows extensibility.